web_widget_dropdown_dynamic/README.rst

```diff
@@ -71,6 +71,13 @@ Usage
        context="{'depending_on': other_field}"
    />
 
+**DEMO**
+
+On User defined filters added new field to show the feature, it is
+called **Dropdown Integer**. If any user selected just One option shoud
+appear, but if Mitchell Admin it should be possible to select option One
+and Two.
```

Comment:
I don't know if its expected, but in a new record, the behaviour doesn't seem to work right:

![image](https://github.com/user-attachments/assets/2a269204-10f1-4b18-af74-df1991ce059e)

Reply:
It's depending on the field "Available for User"

Reply:
But it works when the record is already saved...